### PR TITLE
Feature/cloudwatch

### PR DESCRIPTION
**Pull request by fvazquez-caylent**
Adds new example named minimal-logs.
Adds two variables ingress_protocol and egress_protocol
Updates CHANGELOG.md

### CODE REVIEW

File: examples/minimal-logs/main.tf [hunk 1]
```diff
@@ -0,0 +1,162 @@
+locals {
+  az = "us-east-2a"
```
Comment (franco-martin) [inline examples/minimal-logs/main.tf]: Hardcoding the AZ is not the best, use the datasource for that. 
https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/availability_zones

File: examples/minimal-logs/main.tf [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+}
+
+# Attach CloudWatchAgentServerPolicy to tamr-vm
+resource "aws_iam_role_policy_attachment" "cw-agent-server-policy" {
+  role       = module.tamr-vm.tamr_iam_role.tamr_instance_role_name
+  policy_arn = data.aws_iam_policy.cw-agent-server-policy.arn
+}
+
+# Set up VPC & subnet
+resource "aws_vpc" "tamr_vm_vpc" {
```
Comment (franco-martin) [inline examples/minimal-logs/main.tf:35]: Why not use the vpc module we have?

File: examples/minimal-logs/main.tf [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+module "s3-bucket" {
+  source             = "git::git@github.com:Datatamer/terraform-aws-s3.git?ref=1.1.0"
+  bucket_name        = format("%s-tamr-module-test-bucket", var.name-prefix)
+  read_write_actions = local.tamr_vm_s3_actions
+  read_write_paths   = [""] # r/w policy permitting specified rw actions on entire bucket
+  additional_tags    = var.tags
+}
+
+# Create new EC2 key pair
+resource "tls_private_key" "tamr_ec2_private_key" {
```
Comment (franco-martin) [inline examples/minimal-logs/main.tf:59]: I think it'd be better to input the name of the key as a variable

File: examples/minimal-logs/main.tf [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+  algorithm = "RSA"
+}
+
+module "tamr_ec2_key_pair" {
```
Comment (franco-martin) [inline examples/minimal-logs/main.tf]: Lets use the key pair as a variable

File: examples/minimal-logs/terraform.tfvars [hunk 5]
```diff
@@ -0,0 +1,6 @@
+name-prefix = "poc-cw"
+#ami_id = "ami-0443305dabd4be2bc"
```
Comment (franco-martin) [inline examples/minimal-logs/terraform.tfvars:2]: this should not be here

File: examples/minimal-logs/main.tf (hunk 1 above)
Comment (fvazquez-caylent) [reply examples/minimal-logs/main.tf]: Fixed. Thanks for your input.

File: examples/minimal-logs/main.tf (hunk 3 above)
Comment (fvazquez-caylent) [reply examples/minimal-logs/main.tf:59]: Fixed. Thanks for your input.

File: examples/minimal-logs/main.tf (hunk 4 above)
Comment (fvazquez-caylent) [reply examples/minimal-logs/main.tf]: Fixed.

File: examples/minimal-logs/terraform.tfvars (hunk 5 above)
Comment (fvazquez-caylent) [reply examples/minimal-logs/terraform.tfvars:2]: Fixed.

**Pull request merged**